packages/block/src/header.ts

```diff
@@ -315,9 +316,7 @@ export class BlockHeader {
     } else {
       // pre-homestead
       if (
-        parentTs
-          .addn(this._common.paramByHardfork('pow', 'durationLimit', hardfork))
-          .cmp(blockTs) === 1
+        parentTs.addn(common.paramByHardfork('pow', 'durationLimit', hardfork)).cmp(blockTs) === 1
```

Comment:
small nit, I always find a direct comparison easier to read than `cmp`:

```suggestion
        parentTs.addn(common.paramByHardfork('pow', 'durationLimit', hardfork)).gt(blockTs)
```

Reply:
Just FYI the logic here is not changed (just copying code around), but have fixed this in 34fa3ea because I agree it is easier to understand the code to use gt/lt (all `cmp`s are now removed)